crescent/bot.py

```diff
@@ -1,14 +1,15 @@
 from __future__ import annotations
 
+from asyncio import Event as aio_Event
 from asyncio import Task, create_task
```

Comment:
seems weird to import stuff from asyncio like this. also `aio_Event` also feels weird. Maybe `_Event`, if just importing asyncio isn't an option?

Reply:
Event from Hikari and Asyncio is used here so It would be confusing to use underscore for one of them. And I wanted to keep it as from imports so its consistent with everything else.